playbooks/init/basic_facts.yml

```diff
@@ -16,104 +16,15 @@
     import_role:
       name: openshift_sanitize_inventory
 
-  - name: Detecting Operating System from ostree_booted
-    stat:
-      path: /run/ostree-booted
-      get_checksum: false
-      get_attributes: false
-      get_mime: false
-    register: ostree_booted
+  - name: set openshift_deployment_type if unset
```

Comment:
Wrong comment name?

I think this should be replaced with `fail` task to make sure Atomic hosts cannot be added to the cluster

Reply:
Yeah, wrong task name.  Just a hack because I removed the openshift_is_atomic logic, but then needed to backtrack because that affects things all over the place.